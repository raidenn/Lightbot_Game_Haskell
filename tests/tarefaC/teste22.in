ZZZ
ZAZ
ZZZ
1 1 E
LSL
