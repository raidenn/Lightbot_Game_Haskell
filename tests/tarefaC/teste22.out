1 1
1 1
INCOMPLETO
